### ISSUE TITLE
can't merge

### ISSUE THREAD

**Issue by salsawij** [Olivr/free-domain#519]
it always says this `Make sure that only one line was affected`, how am I suppose to resolve this?

**Comment by redwerkz** [comment]
I think this is because both zone files lack a newline at the end

**Comment by ririko5834** [comment]
I have the same problem

**Comment by py660** [comment]
If they both lack a newline at the end, how are they different?

**Comment by cjabb** [comment]
> I think this is because both zone files lack a newline at the end

yeah that's the issue but it seems even when adding a newline the error still occurs.

**Comment by ririko5834** [comment]
@olivr-bot

**Comment by ririko5834** [comment]
@politician take a look at it

**Comment by py660** [comment]
I found that if you use github.dev, everything works perfectly.

**Comment by ririko5834** [comment]
It didnt work for me there too

**Comment by ririko5834** [comment]
Finally got it, you need to add your line between other lines, not at the end of the file

**Comment by ririko5834** [comment]
Didnt work :( https://github.com/Olivr/free-domain/pull/645/files only 1 line is affected. That check isn't working well, it should be fixed or removed that only 1 line affected checking

**Comment by jaymineh** [comment]
Same issue I'm still having. Can someone look into this please? @olivr-bot @politician
